docroot/modules/custom/uiowa_core/src/Plugin/Block/SkeletonLoadBlock.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Drupal\uiowa_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides an 'Skeleton Load' block.
+ *
+ * @Block(
+ *   id = "skeleton_load_block",
+ *   admin_label = @Translation("Skeleton Load"),
+ *   category = @Translation("Site custom")
```

Comment:
We don't want this block to be placed in Layout Builder directly right? Shouldn't this be `Restricted`?

```suggestion
 *   category = @Translation("Restricted")
```